### PR TITLE
Aws lakeformation

### PR DESCRIPTION
**Pull request by shineyb**

### CODE REVIEW

File: c7n/resources/lakeformation.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+@resources.register('lakeformation')
+class LakeFormation(QueryResourceManager):
+
+    class resource_type(TypeInfo):
+        service = 'lakeformation'
+        enum_spec = ('list_resources', 'ResourceInfoList[].ResourceArn', None)
+        arn = id = 'ResourceArn'
+        name = 'name'
+        detail_spec = ("describe_resource", "ResourceArn", None, "ResourceInfo")
```
Comment (ajkerrigan) [inline c7n/resources/lakeformation.py:22]: Do we need `detail_spec` here? It doesn't seem to return any information we're not getting from `list_resources`.

**Review by ajkerrigan — COMMENTED** [review]
Thanks for the contribution @shineyb ! Just one question inline regarding the `detail_spec`.

File: c7n/resources/lakeformation.py (hunk 1 above)
Comment (shineyb) [reply c7n/resources/lakeformation.py]: @ajkerrigan : Thank you for reviewing ! I have removed detail_spec .

Comment (ajkerrigan) [reply c7n/resources/lakeformation.py]: Great, thanks! 👍

**Review by ajkerrigan — APPROVED** [review]
Thanks again, looks good 😎

**Pull request merged**